resource-sharing-partners/src/test/java/no/sikt/rsp/ResourceSharingPartnerTest.java

```diff
@@ -441,10 +447,61 @@ public void shouldExtractPartnerDetailsProfileDataNncipCorrectlyIgnoringInvalidE
         assertNncipProfileDetailsPopulatedCorrectly(partner, record.getBibnr(), null);
     }
 
+    @Test
+    public void foreignNonAlmaOrBibsysLibrariesWithNncipShouldHavePartnerProfileTypeEmail() throws IOException {
+        final Record record = new RecordBuilder(BigInteger.ONE, LocalDate.now(), BaseBibliotekUtils.KATSYST_TIDEMANN)
+                                  .withBibnr("1000000")
+                                  .withLandkode(BaseBibliotekUtils.COUNTRY_CODE_GERMAN)
```

Comment:
Ok, her er tysk brukt, men bare i test sammenheng. Jeg ville har ført det da i Test klassen som constant, men er det fasit??? Opp til deg.
Den testen ville være enda mer elegant (uavhengig) hvis vi tester det med en random landkode. Strengt tatt tester vi bare to tilfelder: "no" og "de". Litt pirkete det - jeg vet - men hva skjer nå landkode er "fr"?

Reply:
Du har et poeng. Jeg flytter landkoder som ikke brukes i forretningslogikk til test, lager en parametrisert test som tester med flere ikke-norske landkoder.